scripts/core/EXT_EXP_BindlessImages.rst

```diff
@@ -46,7 +47,8 @@ This function only allows for the allocation of image memory in an implementatio
 In this extension, we propose the following additions:
  * Provide a new image descriptor and flags for Bindless images.
  * Support for creation of images on linearly allocated memory backed by USM.
- * Extension API to create an image handle from pitched memory
+ * Extension API to create an image handle from pitched memory.
+ * Create sampled image from the new image descriptor and a sampler.
```

Comment:
can we add a flag indicated this is Sampled image  ? 
somehting like : 
// Indicate we allocate bindless SampledImage
            bindlessExtDesc.flags = ZE_IMAGE_BINDLESS_EXP_FLAG_BINDLESS | ZE_IMAGE_BINDLESS_EXP_FLAG_SAMPLED_IMAGE;